security/cmd/node_agent_k8s/main.go

```diff
@@ -86,6 +86,10 @@ const (
 	// The environmental variable name for key rotation job running interval.
 	// example value format like "20m"
 	SecretRotationJobRunInterval = "SECRET_JOB_RUN_INTERVAL"
+
+	// The environmental variable name for staled connection recyle job running interval.
```

Comment:
s/recyle/recycle, here and everywhere in this pr

Reply:
updated.